engine/runtime-parser/src/main/java/org/enso/compiler/core/ir/DefinitionArgument.java

```diff
@@ -73,6 +73,32 @@ public Specified(Name name) {
       this(name, Option.empty(), Option.empty(), false, null, new MetadataStorage());
     }
 
+    @Override
+    public String showCode(int indent) {
```

Comment:
The method `showCode` should be generated as `abstract` or left abstract from `IR` interface. It has to be implemented otherwise we risk `AbstractMethodError` being thrown at random places that try to work with the `IR` and rely on functional `showCode` being there.

Reply:
Implemented in 2b1432545da6c0c3f5359f4f91921d4732112c19 - implementors are now forced to implement their own `showCode`.